packages/ble-transport/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@aries-framework/ble-transport",
+  "main": "build/index",
+  "types": "build/index",
+  "version": "0.1.0",
+  "files": [
+    "build"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "license": "Apache-2.0",
+  "description": "Bluetooth Low Energy transport for Aries Framework JavaScript",
+  "homepage": "https://github.com/hyperledger/aries-framework-javascript-ext/tree/main/packages/ble-transport",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/hyperledger/aries-framework-javascript-ext",
+    "directory": "packages/ble-transport"
+  },
+  "scripts": {
+    "sample": "tsnd --respawn samples/sample.ts",
+    "build": "yarn run clean && yarn run compile",
+    "clean": "rimraf -rf ./build",
+    "compile": "tsc -p tsconfig.build.json",
+    "prepublishOnly": "yarn run build",
+    "test": "jest"
+  },
+  "dependencies": {
+    "@animo-id/react-native-ble-didcomm": "0.3.0"
```

Comment:
This should be a peer dependency as packages are only auto-linked when they're in the package.json dependencies of the app